ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/port-channel-interfaces.j2

```diff
@@ -97,7 +97,43 @@
 {%            endif %}
 {%        endfor %}
 {%    endif %}
+{# OSPF #}
+{%     set po_interface_ospf = namespace() %}
+{%     set po_interface_ospf.configured = false %}
+{%     for port_channel_interface in port_channel_interfaces | arista.avd.natural_sort %}
+{%          if port_channel_interfaces[port_channel_interface].ospf_area is defined %}
+{%              set po_interface_ospf.configured = true %}
+{%          endif %}
+{%     endfor %}
+{%     set port_channel_interface_ospf = namespace() %}
+{%     set port_channel_interface_ospf.configured = false %}
+{%     if port_channel_interfaces is defined and port_channel_interfaces is not none %}
+{%         for port_channel_interface in port_channel_interfaces | arista.avd.natural_sort %}
+{%              if port_channel_interfaces[port_channel_interface].ospf_area is defined %}
+{%                  set port_channel_interface_ospf.configured = true %}
+{%              endif %}
+{%         endfor %}
+
+{%     endif %}
+{%     if po_interface_ospf.configured == true or port_channel_interface_ospf.configured == true %}
```

Comment:
What is the difference between the two loops above? I think this could be simplified.

Reply:
Fixed in f5627b4